tools/bazar/fields/TagsField.php

```diff
@@ -112,4 +113,24 @@ protected function renderStatic($entry)
             return null ;
         }
     }
+
+    public function getOptions()
+    {
+        if (empty($this->options) || count($this->options) == 0) {
```

Comment:
les deux conditions sont équivalentes mêle si empty est plus lisible...

Reply:
a priori, si $this->options[0] = null, empty renvoit false alors que count renvoie 1, ... non ? Je peux me tromper.

Reply:
idem si $this->options[0] = false

Reply:
je te confirme que c'est équivalent. empty est plus rapide à exécuter toutefois mais surtout plus lisible

Reply:
Super merci pour le conseil, je corrige et je fusionne